CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/GlyphRectangleBuffer.java

```diff
@@ -356,4 +357,14 @@ private boolean bufferedImagesEqual(BufferedImage img1, BufferedImage img2) {
         }
         return true;
     }
+
+    private synchronized Integer synchronizedAddHashcode(int hashCode, BufferedImage img, int extra, int w, int h) {
```

Comment:
return type can be primitive `int` and all parameters can be final

Reply:
Done